wraith/bin/wraith.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+read -p "Specify the port number that your local dev environment is running on: " port
+
+port=${port:-5000}
```

Comment:
This is what I had in the first place, but @benbarnett said the warnings were silent on the fact that it was running on a different port